lib/Container/CoreExtension.php

```diff
@@ -80,7 +80,7 @@ private function getBaseCwd($path = null) {
         }
 
         // Return base CWD where .git directory is present
-        if (!file_exists(sprintf('%s/.git', $path)) && $path !== '/') {
+        if (!file_exists(sprintf('%s/.git', $path)) && $path !== '/' || strpos($path, 'vendor')) {
```

Comment:
vendor is configurable in `composer.yml`, it is not guaranteed to be `vendor`.

Reply:
Yes, but you supposed that is `vendor` because you hardcoded the `autoload` attributes (https://github.com/phpactor/phpactor/blob/master/lib/Container/CoreExtension.php#L59)

Reply:
Thats configurable :)

Reply:
Is there a usercase that overrides the `autoload` without overriding `cwd` ?

Reply:
In this part of configuration, we supposed the user using default configuration. If the `composer.json` file is overridden, that means the user must also define an override file, no ?

Reply:
Yeah, you can actually configure mutliple autoloaders (e.g. some applications have their own package management).

Also, some people change the name of `vendor` in `composer.json`

Reply:
Are you checking this entry somewhere ? Or the user must override the default configuration to performs `autoloader` and `cwd` ?

Reply:
> Are you checking this entry somewhere ?

Not sure what you mean. If the `vendor/autoload.php` (of whatever is configured) is not found, an exception will be thrown.

Reply:
So, for a default configuration, what did you mean about vendor directory could change ?

Reply:
```

Reply:
See https://getcomposer.org/doc/06-config.md#vendor-dir.

I think potentially a better solution to this issue is to always pass the `cwd` as an option to Phpactor from the plugin, so that it is _always_ the path in which VIM was loaded.

Would that make sense?